### ISSUE TITLE
Cannot Log-in: FFXI-ERROR:3001

### ISSUE THREAD

**Issue by nfontanait** [EdenServer/community#5951]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

**Comment by nfontanait** [comment]
Hi, I was in my Mog House, in Bastok Mines, and chose the option to exit to Port Bastok. The game stayed on the black loading screen between zones and eventually timed out. I am not able to log-in as that character anymore, FFXI-Error: 3001. I am however able to log-in as a different character on the same account. Is there anyone who can assist? 

Edit: I have tried waiting 5-10 minutes, several times. I have tried restarting PC. I have tried deleting the "USER" folder.

Character: Everlost
